src/main/resources/db/migration/V1.3__add_municipality_id.sql

```diff
@@ -0,0 +1,12 @@
+alter table if exists disturbance
+   add column municipality_id varchar(255) AFTER id;
+   
+alter table if exists subscription
+   add column municipality_id varchar(255) AFTER id;
+   
+create index municipality_id_index 
+   on disturbance (municipality_id);
+    
+create index municipality_id_index 
+   on subscription (municipality_id);
```

Comment:
I think index names have to be unique.

Reply:
They are unique, since they belong to different tables. Each table has it's own "namespace" and the qualified name will be <tablename>.<indexname>

See example:
![image](https://github.com/user-attachments/assets/952bf187-ed1d-4935-a528-e258850ab3ef)